verification/spec/relayer/ChannelHandlers.tla

```diff
@@ -0,0 +1,172 @@
+-------------------------- MODULE ChannelHandlers --------------------------
+
+(***************************************************************************
+ This module contains definitions of operators that are used to handle
+ channel datagrams
+ ***************************************************************************)
+
+EXTENDS Naturals, FiniteSets
+
+ChannelIDs == {"chanAtoB", "chanBtoA"}
+nullChannelID == "none"
+
+ChannelStates == {"UNINIT", "INIT", "TRYOPEN", "OPEN", "CLOSED"}
+ChannelOrder == {"ORDERED", "UNORDERED"}
+
+(***************************************************************************
+ Channel helper operators
+ ***************************************************************************)
+
+\* get the channel ID of the channel end at the connection end of chainID
+GetChannelID(chainID) ==
+    IF chainID = "chainA"
+    THEN "chanAtoB"
+    ELSE IF chainID = "chainB"
+         THEN "chanBtoA"
+         ELSE nullChannelID
+         
+\* get the channel ID of the channel end at chainID's counterparty chain
+GetCounterpartyChannelID(chainID) ==
+    IF chainID = "chainA"
+    THEN "chanBtoA"
+    ELSE IF chainID = "chainB"
+         THEN "chanAtoB"
+         ELSE nullChannelID          
+
+(***************************************************************************
+ Channel datagram handlers
+ ***************************************************************************)
+ 
+\* Handle "ChanOpenInit" datagrams
+HandleChanOpenInit(chainID, chain, datagrams) ==
+    \* get "ChanOpenInit" datagrams, with a valid channel ID
+    LET chanOpenInitDgrs == {dgr \in datagrams : 
+                            /\ dgr.type = "ChanOpenInit"
+                            /\ dgr.channelID = GetChannelID(chainID)} IN
+    
+    IF chanOpenInitDgrs /= {} /\ chain.connectionEnd.state /= "UNINIT"
+    \* if there are valid "ChanOpenInit" datagrams and the connection is not "UNINIT", 
+    \* create a new channel end and update the chain
```

Comment:
Can you move the comment above the `IF`. The comments should refer to lines after the comments. There are a number of places where they are after.